main.py

```diff
@@ -213,6 +214,29 @@ def handle_link_down(self, event):
             if not evc.is_affected_by_link(event.link):
                 evc.handle_link_down()
 
+    @listen_to('kytos/of_core.switch.port.created')
```

Comment:
MEF-Eline should not listen to "of_core" events. We are trying to create a highlevel arquitecture. So let's try to listen to events from 'topology'.

Reply:
Anyway, this should be moved to kytos "core".

Reply:
@beraldoleal , agreed.

By analyzing the code, whenever the controller first create a switch there are no interfaces. Currently, `kytos/of_core` is the first one who'll update the Interface attributes. So, I've patched `kytos.core.controller` to also parse the vlan settings from `kytos.conf` and in addition to create a `Switch()` it'll also create `Interface()`s whenever a particular switch has the configuration in place. And then, I'll have to patch `kytos/of_core` not to completely overwrite an `Interface()` object when it's updating/creating new `Interface()`s.

But, before I push this code upstream, I need you to first merge this PR:

https://github.com/kytos/kytos/pull/836